.travis.yml

```diff
@@ -43,6 +43,14 @@ jobs:
       before_install: bash _ci/init.sh
       script: bash _ci/whitesource.sh
 
+    - name: "Source Clear Scan"
+      stage: build
+      if: branch NOT IN (company_release)
+      before_install: bash _ci/init.sh
+      script: skip
```

Comment:
Can we document somewhere (JIRA, Confluence, README ?) how this is configured ?  For example, policies that will cause the build to fail ?

Reply:
I've added a few details here. Thanks for review.